pynetbox-stubs/_gen/plugins.pyi

```diff
@@ -0,0 +1,234 @@
+
+from typing import Any, Dict, Iterable, List, Optional, Union, overload
```

Comment:
do you want to run black on this file?

Reply:
No I don't - I have fixed the "order of operations" in the latest commit.